designs/virtualization/4.x/create-vm-dropdowns/create-vm-dropdowns.md

```diff
@@ -0,0 +1,117 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# create VM dropdowns
+
+This item will cover various updates to the VM creation wizard
+- Adding descriptions to the dropdown fields (using this pattern https://github.com/patternfly/patternfly-design-kit/issues/218)
+- Renaming fields
+- Reordering fields
+- Adding 'Operating System source' step
+
+
+![Create VM - General-default](img/1-0.jpg)
+Landing on the General step, Default mode.
+Section titles added, and fields reorgenized and renamed.
+Any OS image source related fields are mofed to the next step.
+
+'General' section:
+- 'Name'
+- 'Description'
+'Resources & configuration' section'
+- 'Operating system'
+- 'Workload type' (previously Workload profile)
+- 'Flavor (CPU and Memory)' (previously only 'flavor')
+
+
+## General step
+
+![Create VM - General](img/1-1.jpg)
+Upon selecting an operating system, the Workload type and Flavor (CPU and Memory) fields will be enabled and their defaults will be selected.
+
+![Create VM - General - Workload type](img/1-1-1.jpg)
+Workload type options
+Server is the default
+
+![Create VM - General - flavor](img/1-1-2.jpg)
+Flavor options.
+Medium is the default
+
+
+## Operating system source step (new)
+
+![Create VM - General](img/1-2.jpg)
+Operating system source step,landing mode.
+
+![Create VM - General](img/1-2-1.jpg)
+'Boot operating system via' field options
+
+![Create VM - General](img/1-2-2.jpg)
+Selecting an option from the 'Boot operating system via' field will expose below all the fields required to upload the image and create the correlating disk/NIC.
+For example: on selecting the 'URL' option, additional fields will appear below.
+These fieldss are required in order to point to the desired URL asl well as creating a disk via PVC that will containt the image on that URL.
+The fields are basically mapped 1:1 to the Disk modal, with the URL option selected for 'Disk content' field.
```

Comment:
Changed the images to display the same field name and order.
Please note that the fields order and names did change within the Disk/NIC modal and this image now reflects that.
Also, because the "Boot operating system via*" field is equivalent to "Disk content, but not the same (PXE is available too) I offer that the following fields will be identical to the ones appear in the disk modal, only that the Disk Content is hidden.

Reply:
ok, we could distinguish the "Boot operating system via*" since it will not be part of the modal component anyway. And is the only part which is different. And we will hide the Disk content (source) in the disk modal